lib/new_relic/tracer/macro.ex

```diff
@@ -223,6 +223,12 @@ defmodule NewRelic.Tracer.Macro do
     Macro.postwalk(args, &rewrite_call_term/1)
   end
 
+  # Unwrap Structs into Maps when reporting
+  # They can't always be re-referenced directly since they can have enforced_keys
+  def rewrite_call_term({:%, _, [{:__aliases__, _, _} = struct, {:%{}, line, members}]}) do
```

Comment:
Had to play around in `iex`, but to make sure I understand: this function works because the first value of that tuple would be `:%{}` if this was a map, vs `:%` when it's a struct, right?

Reply:
Yeah, it's basically unwrapping a struct literal into plain map.

It's data that's only used for reporting so it's not changing any aspect of the function definition itself.